kevm-pyk/src/kevm_pyk/kproj/evm-semantics/evm.md

```diff
@@ -1571,109 +918,17 @@ For each `CALL*` operation, we make a corresponding call to `#call` and a state-
 -   `#hasValidInitCode` checks the length of the transaction data in a create transaction. [EIP-3860]
 
 ```k
-    syntax InternalOp ::= "#create"   Int Int Int Bytes
-                        | "#mkCreate" Int Int Int Bytes
-                        | "#incrementNonce" Int
-                        | "#checkCreate" Int Int
- // --------------------------------------------
```

Comment:
Same question here: the API is for sure responsible for creating, incrementing, and checking everything, right?